### DISCUSSION TITLE
Proposal:  <podcast:valuePercentage>

### DISCUSSION THREAD

**Discussion by agates** [Podcastindex-org/podcast-namespace#606] [Tag Proposal]
## Value Percentage
`<podcast:valuePercentage>`<br><br>
This elements specifies a percentage of value splits in combination with `<podcast:remoteItem>` tags that are *not* under `<podcast:valueTimeSplit>`.

This allows a feed to request a percentage of value splits without having to specify specific times for every `<podcast:remoteItem>` in their feed.  This is particularly useful for playlists where the curator asks for a percentage for playlist curation or references in `<podcast:podroll>`, incentivizing discoverability methods.

The `maximumRemotePercentage` attribute from remote `<podcast:value>` blocks *should* be respected.

### Parent
&nbsp; `<podcast:value>` or `<podcast:remoteItem>`

### Count
&nbsp; Single

### Node Value
&nbsp; A number from 0 to 100, indicating a percentage.  If not defined, defaults to 0.  If the value is less than 0, 0 is assumed.  
  If the value is greater than 100, 100 is assumed.

### Example (Music Playlist requesting 5% of the value at the channel, but 1% for one remoteItem)
```xml
<rss xmlns:podcast="https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd" version="2.0">
    <channel>
        <title>Picking the Hits 2.0!</title>
        <description>All the hits played on the Podcasting 2.0 show.</description>
        <link>https://podcastindex.org</link>
        <language>en-US</language>
        <pubDate>Wed, 07 Jun 2023 04:30:38 GMT</pubDate>
        <image>
            <url>https://example.com/images/pci_avatar-massive.jpg</url>
        </image>

        <podcast:guid>3f2a8e4e-263a-51aa-9d3d-0d71f82a1564</podcast:guid>
        <podcast:medium>musicL</podcast:medium>
        <itunes:image>https://example.com/images/pci_avatar-massive.jpg</itunes:image>

        <podcast:value type="lightning" method="keysend" suggested="0.00000005000">
            <podcast:valuePercentage>5</podcast:valuePercentage>
            <podcast:valueRecipient name="podcaster" type="node" address="036557ea56b3b86f08be31bcd2557cae8021b0e3a9413f0c0e52625c6696972e57" split="99" />
            <podcast:valueRecipient name="hosting company" type="node" address="036557ea56b3b86f08be31bcd2557cae8021b0e3a9413f0c0e52625c6696972e57" split="1" />
        </podcast:value>

        <podcast:remoteItem
                feedGuid="ff519475-6e90-5231-91a0-37d092088d88"
                feedUrl="https://media.rss.com/joemartinmusic/feed.xml"
                itemGuid="e75771b1-e8d4-4133-9392-c579822247d9"
                medium="music"
        >
            <podcast:valuePercentage>1</podcast:valuePercentage>
        </podcast:remoteItem>
        
        <podcast:remoteItem
                feedGuid="47081700-bd65-511f-b535-f545f3cd660c"
                feedUrl="https://player.wavlake.com/feed/d1ed0ec9-21a8-4eda-b2c9-b17c8019a7e8"
                itemGuid="7b03666e-b323-499d-93a7-ca51ce627ffd"
                medium="music"
        />

       <podcast:remoteItem
               feedGuid="483dde8e-7e94-59a7-8eb0-2b0dc64a87bd"
               feedUrl="https://player.wavlake.com/feed/1dd1bbd8-1084-4fdc-9788-dddaa62fbc6a"
               itemGuid="8501fb64-a6a3-475a-8b10-9c746f0fe579"
               medium="music"
       />

       <podcast:remoteItem
               feedGuid="b40ffcf7-2c48-5cfe-8daa-b65d766b2c25"
               feedUrl="https://www.wavlake.com/feed/92b04241-97f5-4ff7-be11-cf45f70812e7"
               itemGuid="9a48aab8-6da6-4cc1-9951-5b049c333580"
               medium="music"
       />
    </channel>
</rss>
```

**Comment by samsethi** [comment]
Thanks @agates we built this function into TrueFans. We give 1% of sats earned to the playlist curator.  We can change this default amount in our admin dashboard.  We will look at this proposal as this sounds like a user generated split request which we can add to the users settings.  We built our functionality using VTS.  So we honour the individual wallet splits for each episode but apply the 1%.    

<img width="1135" alt="Cursor_and_TrueFans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/63333c97-0dc5-45f4-8740-c3520ba63d9b">

**Comment by agates** [comment]
In case it wasn't obvious this depends on https://github.com/Podcastindex-org/podcast-namespace/discussions/607
